src/frontend/src/sol/services/sol-balance.services.ts

```diff
@@ -0,0 +1,11 @@
+import { SOLANA_KEY_ID } from '$env/networks/networks.sol.env';
+import { getSchnorrPublicKey } from '$lib/api/signer.api';
+import type { CanisterApiFunctionParams } from '$lib/types/canister';
+
+export const getSolanaPublicKey = async (
+	params: CanisterApiFunctionParams<{ derivationPath: string[] }>
+): Promise<Uint8Array | number[]> =>
```

Comment:
This should prepend "SOL" to the derivation path.

Reply:
like this? https://github.com/dfinity/oisy-wallet/pull/3925/commits/6974e3852c09742e758aaf872637345781ca2f1c